.nais/vars-p.yaml

```diff
@@ -7,3 +7,6 @@ ingresses:
   - "https://meldekortservice.intern.nav.no"
   - "https://meldekortservice.nais.adeo.no"
   - "https://app.adeo.no/meldekortservice"
+  - "https://meldekortservice.prod-fss-pub.nais.io"
```

Comment:
Må åpnes for sonekryssing, ref. https://doc.nais.io/clusters/migrating-to-gcp/#how-do-i-reach-an-application-found-on-premises-from-my-application-in-gcp

Reply:
Done